lib/src/config/introduction.dart

```diff
@@ -21,6 +21,18 @@ enum IntroductionDisplayMode {
   multiPageHorizontal
 }
 
+enum IntroductionControlMode {
+  twoButton,
```

Comment:
maybe change this to singleButton and previousNextButton, as it more clearly indicates the purpose of the control modes. Other than that good commit